setup.py

```diff
@@ -66,6 +66,13 @@ def read_main_file(key):
             if key in line:
                 return line.split('"')[1]
 
+entries = {"pycbc.waveform.td":
+            ["GWS-NRHybSur3dq8 = gwsurrogate.pycbc:gws_td_gen",
+             "GWS-NRSur7dq4 = gwsurrogate.pycbc:gws_td_gen",
+             "GWS-NRHybSur3dq8Tidal = gwsurrogate.pycbc:gws_td_gen",
+            ]
+          }
```

Comment:
@ahnitz: Sorry for the extreme delay in looking at this :)
I was wondering: Would it be better to do this interfacing on the pycbc side?
One issue I foresee is that someone will need to remember to add to this list as new models get added.
Instead, from the pycbc side, assuming gwsur is installed, can't you just get the list of model like [here](https://github.com/sxs-collaboration/gwsurrogate#available-models) and do something similar?
That way, no new maintenance should be required?

Reply:
@vijayvarma392 Swooping in here… if there is a list of approximants you could just do this programmatically in the setup file. That should alleviate concerns about remembering to update.

Reply:
Yes, I think that can be done. But I guess more fundamental question is where is the natural place to host this: pycbc or gwsur?

Reply:
I'm on vacation atm, so may not get back to this in the next couple weeks unfortunately. My two cents though. I think @cdcapano has the right approach. @vijayvarma392 Your question is a good one. I think it makes more sense to have that on the gwsur side, but our goals is to be as useful as possible in terms of interoperability. 

This sort of things comes up with other packages we are trying to interface to as well. One issue that has been raised elsewhere is that developers don't want to have to edit pycbc to say use their new waveform model in our codes. Our goal is to provide stable interfaces, so that people can hook into the analysis functionality, but we don't control what sort of models people might develop. The thinking is that it then make sense to keep the choice of 'exported' approximants close to where they are actually implemented. It's more likely to get out of sync on the other end (if it's a manual list). If the list is programmatic, then I think Collin's approach is sensible if possible.